### DISCUSSION TITLE
Stuck on building

### DISCUSSION THREAD

**Discussion by loopdee43** [community/community#42584] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My github pages at [url](https://github.com/loopdee43/UnityProject/) has been building with the message "Your GitHub Pages site is currently being built from the main branch" for the past hour and a half. I've read its only supposed to take 20 minutes max, so I waited that, and then some more... Anyone know why this is happening? Any help would be appreciated.

**Comment by radar-blip** [comment, marked as answer]
Not sure if this is causing the problem, but look here:

![Screenshot from 2022-12-24 23-37-49](https://user-images.githubusercontent.com/39196767/209457159-f17c7ee7-4c30-4ba1-b0be-740af52291d7.png)

Found by looking at your commit here:
![Screenshot from 2022-12-24 23-43-17](https://user-images.githubusercontent.com/39196767/209457224-1d9cdf36-9ce3-4b44-9f2a-276518bd1d9f.png)

**Reply by loopdee43** [reply]
Yup that was it. Turns out I actually don't necessarily need that directory anyways so thats on me. Thanks!

**Reply by radar-blip** [reply]
You're welcome! 👍🏻
